mempool/src/core_mempool/transaction.rs

```diff
@@ -86,8 +100,8 @@ pub enum TimelineState {
 
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
 pub struct SequenceInfo {
-    pub transaction_sequence_number: u64,
-    pub account_sequence_number: u64,
+    pub transaction_replay_protector: ReplayProtector,
+    pub account_sequence_number: AccountSequenceNumberInfo,
```

Comment:
I don't think we need this in the MempoolTransaction itself. It should just be passed in as a separate argument when calling insert. (I was thinking the same while looking at Aaron's PR as well.

Removing it, we can remove SequenceInfo and just directly use ReplayProtector.

Reply:
Resolved with https://github.com/aptos-labs/aptos-core/pull/16404

Reply:
Done.